lib/superset/services/import_dashboard_across_environment.rb

```diff
@@ -78,6 +80,10 @@ def update_dataset_configs
         dashboard_config[:datasets].each do |dataset|
           dataset[:content][:database_uuid] = dashboard_config[:databases].first[:content][:uuid]
           dataset[:content][:schema]        = target_database_schema
+
+          # by clearing out the source's catalog, this automatically allows superset to set to the default catalog on the target
+          dataset[:content][:catalog]       = nil
```

Comment:
Note .. at RDY we do not use catalogs atm for high level namespace architecture
so for our purposes currently nil is fine.